libraries/AP_Scripting/examples/ahrs-set-origin.lua

```diff
@@ -0,0 +1,16 @@
+-- example script for using "set_origin()"" and "initialized()"
+-- sets the ekf origin if not already set
+
+
+function update ()
+
+    if ahrs:initialised() then
```

Comment:
Could do an early return
```suggestion
     if not ahrs:initialised() then
        return update, 5000
    end
```